Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -71,6 +71,21 @@ public override void Update(float frameTime)
         }
     }
 
+    private int GetCurrentGrowthStage(Entity<PlantHolderComponent> entity)
+    {
+        var (uid, component) = entity;
+
+        var age = component.Age;
+
+        if (component.Seed == null)
+            return 0;
+
+        var maturation = component.Seed.Maturation;
+        var stages = component.Seed.GrowthStages;
```

Comment:
These don't have to be re-declared as local variables.

Reply:
Ah, right, that's from when I was trying to catch where the null reference issue was.